src/utils/peer.ts

```diff
@@ -49,17 +49,19 @@ effect(() => {
     .on("open", () => handleNonHostPeerOpened(p));
 });
 
+// for host to handle incoming connection from clients
```

Comment:
Can you also add something like `// TODO use a better name` for this one and the later one? The `TODO`s can be tracked.